pkg/operator/ceph/cluster/rbd/controller.go

```diff
@@ -188,6 +189,10 @@ func (r *ReconcileCephRBDMirror) reconcile(request reconcile.Request) (reconcile
 	daemon := string(opconfig.MonType)
 	currentCephVersion, err := cephclient.LeastUptodateDaemonVersion(r.context, r.clusterInfo, daemon)
 	if err != nil {
+		if strings.Contains(err.Error(), "error calling conf_read_file") {
```

Comment:
Shall we define this string as a const since we use it so many places?

Reply:
I was wondering the same, I guess it does not hurt, I can do that.